Cargo.toml

```diff
@@ -49,6 +49,7 @@ headless_chrome = "1"
 urlencoding = "2.1"
 scraper = "0.19"
 futures = "0.3"
+moka = { version = "0.12.10", features = ["future"] }
```

Comment:
Moka is a cache system. The idea is that when parsing images, we async read from the file system then cache the image in moka, so when picolauncher requests for another image, no file system read is needed and the image can be loaded quickly.